### ISSUE TITLE
Allow for logging after opponent rage quits

### ISSUE THREAD

**Issue by MorgothNL** [stronglink/crucibletracker-issues#26]
I would really like to see the possibility of 'report as win' still be available when an opponent just leaves without conceding first. The win/loss tracking of my decks isn't close to accurate because at least 25% of my 'wins' are people just leaving, which wont get logged as win. 
So just make the report button stay if there was no concede or win/loss?

**Comment by feydreva** [comment]
I agree !!!! same issue here.. I have 25% winrate because opposent rage quit and do not concede the game
